Formula/metabat.rb

```diff
@@ -0,0 +1,80 @@
+class Metabat < Formula
+  # cite Kang_2015: "https://doi.org/10.7717/peerj.1165"
+  desc "Statistical framework for reconstructing genomes from metagenomic data"
+  homepage "https://bitbucket.org/berkeleylab/metabat/"
+  url "https://bitbucket.org/berkeleylab/metabat/get/v2.13.tar.gz"
+  sha256 "aa75a2b62ec9588add4c288993821bab5312a83b1259ff0d508c215133492d74"
+  head "https://bitbucket.org/berkeleylab/metabat.git"
+
+  depends_on :linux unless build.head?
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "cmake" => :build
+
+  fails_with :clang # needs OpenMP
+
+  if OS.mac?
```

Comment:
Please add a one-line comment here explaining that `boost` needs to be compiled with GCC.

Reply:
Good to merge after that. Thank you, Gabriel!